pkgs/development/tools/sigrok-firmware-fx2lafw/default.nix

```diff
@@ -0,0 +1,33 @@
+{ lib
+, stdenv
+, fetchurl
+, sdcc
+}: stdenv.mkDerivation (finalAttrs: {
+  pname = "sigrok-firmware-fx2lafw";
+  version = "0.1.7";
+
+  src = fetchurl {
+    url = with finalAttrs; "https://sigrok.org/download/source/${pname}/${pname}-${version}.tar.gz";
```

Comment:
```suggestion
    url = "https://sigrok.org/download/source/sigrok-firmware-fx2lafw/sigrok-firmware-fx2lafw-${version}.tar.gz";
```

Reply:
Why just `${version}` and not `${pname}`?

Reply:
because version often changes when updating but not pname.